strategy-cache-only/service-worker.js

```diff
@@ -0,0 +1,34 @@
+var CACHE = 'cache-only';
+
+// On install, cache some resource.
+self.addEventListener('install', function(evt) {
+  console.log('The service worker is being installed.');
+  evt.waitUntil(precache());
+});
+
+// On fetch, use cache only strategy.
+self.addEventListener('fetch', function(evt) {
+  console.log('The service worker is serving the asset.');
+  evt.respondWith(fromCache(evt.request));
+});
+
+// Open a cache and use `addAll()` with an array of assets to add all of them
+// to the cache. Extend the installation until all the assets are saved.
```

Comment:
what does 'extend the installation' mean? do you mean the installation isn't finished until the assets are saved to the cache?